mcp/src/graph/graph.ts

```diff
@@ -52,6 +52,17 @@ export async function search(
     return toNodes(result, concise, output);
   }
 }
+export async function get_rules_files(concise: boolean = false) {
```

Comment:
we dont need JSON like this, it should be just a bunch of text, with just the Filename and the content